data/human/hails.txt

```diff
@@ -1964,6 +1964,123 @@ phrase "friendly civilian"
 	word
 		"."
 
+phrase "friendly civilian"
+	word
+		"Sometimes it's too good to be true. "
+		"Don't take unnecessary risks. "
+		"Don't get on the bad side of the authorities. "
+		"I always try to be an honest captain. "
+		"Crime doesn't pay in the end. "
+		""
+	word
+		"It's a fact"
+		"I've heard"
+		"I heard a rumor"
+		"You should know"
+		"They say"
+		"It's said"
+	word
+		" that "
+	word
+		"smugglers almost always get caught in the end."
+		"even the most powerful warship won't save you from a knife in the back."
+		"the Quarg will arrest you if they catch a whiff of any of their equipment on your ship."
+		"the Quarg imprison anyone they catch carrying their outfits."
+		"nine out of ten pirates die within their first year."
```

Comment:
```suggestion
		"nine out of ten pirates die within their first year."
		"Republic Intelligence has their eyes everywhere."
		"the law will always catch up to those who try to bypass it."
```
I want to think up of more, but to be honest, there isn't very many illegal things you can do.

Reply:
This is starting to seem a lot less my PR... :)
Thanks for the suggestions, though.